Makefile

```diff
@@ -1,49 +1,192 @@
 # Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
 # See LICENSE.txt for license information.
 
+################################################################################
+##                             VERSION PARAMS                                 ##
+################################################################################
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.20
```

Comment:
One more

Reply:
@gabrieljackson Do you mean to bump to 1.21?

Reply:
Sorry, lost my first comment on this. Yes I was referring to bumping to v1.21 to keep things in sync.